bugg (java.lagg.string)

"attempt to invoke virtual method 'java.lag.string org.jsoup.nodes.Node.attr(java.lang.String)' on a null object reference"

This message pops up and it refuse to open any mangas doesnt matter with one I try. restart and and clear cookies doesnt help.

Hey !
Here is the stacktrace : 
```
java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String org.jsoup.nodes.Node.attr(java.lang.String)' on a null object reference
    at eu.kanade.tachiyomi.source.online.english.Readmangatoday.imageUrlParse(Readmangatoday.kt:164)
    at eu.kanade.tachiyomi.source.online.english.Readmangatoday.pageListParse(Readmangatoday.kt:160)
    at eu.kanade.tachiyomi.source.online.ParsedHttpSource.pageListParse(ParsedHttpSource.kt:175)
    at eu.kanade.tachiyomi.source.online.HttpSource$fetchPageList$1.call(HttpSource.kt:243)
    at eu.kanade.tachiyomi.source.online.HttpSource$fetchPageList$1.call(HttpSource.kt:24)
    at rx.internal.operators.OnSubscribeMap$MapSubscriber.onNext(OnSubscribeMap.java:69)
    at rx.internal.operators.OnSubscribeDoOnEach$DoOnEachSubscriber.onNext(OnSubscribeDoOnEach.java:101)
    at eu.kanade.tachiyomi.network.OkHttpExtensionsKt$asObservable$1$requestArbiter$1.request(OkHttpExtensions.kt:25)
    at rx.internal.producers.ProducerArbiter.setProducer(ProducerArbiter.java:126)
    at rx.internal.operators.OperatorOnErrorResumeNextViaFunction$4$1.setProducer(OperatorOnErrorResumeNextViaFunction.java:130)
    at rx.internal.operators.OnSubscribeMap$MapSubscriber.setProducer(OnSubscribeMap.java:102)
    at rx.Subscriber.setProducer(Subscriber.java:205)
    at eu.kanade.tachiyomi.network.OkHttpExtensionsKt$asObservable$1.call(OkHttpExtensions.kt:45)
    at eu.kanade.tachiyomi.network.OkHttpExtensionsKt$asObservable$1.call(OkHttpExtensions.kt:13)
    at rx.Observable.unsafeSubscribe(Observable.java:10256)
    at rx.internal.operators.OnSubscribeDoOnEach.call(OnSubscribeDoOnEach.java:41)
    at rx.internal.operators.OnSubscribeDoOnEach.call(OnSubscribeDoOnEach.java:30)
    at rx.Observable.unsafeSubscribe(Observable.java:10256)
    at rx.internal.operators.OnSubscribeMap.call(OnSubscribeMap.java:48)
    at rx.internal.operators.OnSubscribeMap.call(OnSubscribeMap.java:33)
    at rx.Observable.unsafeSubscribe(Observable.java:10256)
    at rx.internal.operators.OperatorOnErrorResumeNextViaFunction$4.onError(OperatorOnErrorResumeNextViaFunction.java:142)
    at rx.exceptions.Exceptions.throwOrReport(Exceptions.java:212)
    at rx.internal.operators.OnSubscribeFromCallable.call(OnSubscribeFromCallable.java:50)
    at rx.internal.operators.OnSubscribeFromCallable.call(OnSubscribeFromCallable.java:33)
    at rx.internal.operators.OnSubscribeLift.call(OnSubscribeLift.java:48)
    at rx.internal.operators.OnSubscribeLift.call(OnSubscribeLift.java:30)
    at rx.Observable.unsafeSubscribe(Observable.java:10256)
    at rx.internal.util.ScalarSynchronousObservable$3.call(ScalarSynchronousObservable.java:235)
    at rx.internal.util.ScalarSynchronousObservable$3.call(ScalarSynchronousObservable.java:228)
    at rx.Observable.unsafeSubscribe(Observable.java:10256)
    at rx.internal.operators.OnSubscribeDoOnEach.call(OnSubscribeDoOnEach.java:41)
    at rx.internal.operators.OnSubscribeDoOnEach.call(OnSubscribeDoOnEach.java:30)
    at rx.Observable.unsafeSubscribe(Observable.java:10256)
    at rx.internal.util.ScalarSynchronousObservable$3.call(ScalarSynchronousObservable.java:235)
    at rx.internal.util.ScalarSynchronousObservable$3.call(ScalarSynchronousObservable.java:228)
    at rx.Observable.unsafeSubscribe(Observable.java:10256)
    at rx.internal.operators.OnSubscribeDoOnEach.call(OnSubscribeDoOnEach.java:41)
    at rx.internal.operators.OnSubscribeDoOnEach.call(OnSubscribeDoOnEach.java:30)
    at rx.Observable.unsafeSubscribe(Observable.java:10256)
    at rx.internal.operators.OnSubscribeMap.call(OnSubscribeMap.java:48)
    at rx.internal.operators.OnSubscribeMap.call(OnSubscribeMap.java:33)
    at rx.Observable.unsafeSubscribe(Observable.java:10256)
    at rx.internal.operators.OperatorSubscribeOn$SubscribeOnSubscriber.call(OperatorSubscribeOn.java:100)
    at rx.internal.schedulers.CachedThreadScheduler$EventLoopWorker$1.call(CachedThreadScheduler.java:230)
    at rx.internal.schedulers.ScheduledAction.run(ScheduledAction.java:55)
    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:428)
    at java.util.concurrent.FutureTask.run(FutureTask.java:237)
    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:272)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
    at java.lang.Thread.run(Thread.java:761)
Caused by: rx.exceptions.OnErrorThrowable$OnNextValue: OnError while emitting onNext value: okhttp3.Response.class
    at rx.internal.operators.OnSubscribeMap$MapSubscriber.onNext(OnSubscribeMap.java:73)
	... 46 more
```

ReadmangaToday changed the class for the image in their reader. I will create a PR soon. 
@inorichi I presume I have to create a PR here and on https://github.com/inorichi/tachiyomi-extensions
Also, to not wait the next release, if I install readmangatoday as a standalone, it will use standalone or the built-in parser?

--- a/app/src/main/java/eu/kanade/tachiyomi/source/online/english/Readmangatoday.kt
+++ b/app/src/main/java/eu/kanade/tachiyomi/source/online/english/Readmangatoday.kt
@@ -161,7 +161,7 @@ class Readmangatoday : ParsedHttpSource() {
         return pages
     }
 
-    override fun imageUrlParse(document: Document) = document.select("img.img-responsive-2").first().attr("src")
+    override fun imageUrlParse(document: Document) = document.select("#chapter_img").first().attr("src")
 
     private class Status : Filter.TriState("Completed")
     private class Genre(name: String, val id: Int) : Filter.TriState(name)

